<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>multiHMCsetWidget</class>
 <widget class="QWidget" name="multiHMCsetWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox_46">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>391</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>罗盘校准</string>
   </property>
   <widget class="QLabel" name="label_82">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>22</y>
      <width>48</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>地磁偏角</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_PSIOFF">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>22</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_48">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>391</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>校准</string>
    </property>
    <widget class="QPushButton" name="pushButton_CALSTART">
     <property name="geometry">
      <rect>
       <x>31</x>
       <y>111</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>开始校准</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_CALEND">
     <property name="geometry">
      <rect>
       <x>98</x>
       <y>111</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>校准完成</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_CALASK">
     <property name="geometry">
      <rect>
       <x>299</x>
       <y>111</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>数据查询</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_CALSTOP">
     <property name="geometry">
      <rect>
       <x>165</x>
       <y>111</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>取消校准</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>X增益：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_KX">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_25">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Y增益：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_KY">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_26">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>X偏差：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_BX">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>30</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_27">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>60</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Y偏差：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_BY">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>60</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_CALSave">
     <property name="geometry">
      <rect>
       <x>232</x>
       <y>111</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>保存校准</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_PSIOFFASK">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>查询</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_PSIOFFSET">
    <property name="geometry">
     <rect>
      <x>207</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>设置</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>391</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>罗盘安装设置</string>
   </property>
   <widget class="QLabel" name="label_83">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>27</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>安装方向：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_HMCDIR">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>25</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>X前  Y右</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>X前  Y左</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>X后  Y右</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>X后  Y左</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>X右  Y前</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>X左  Y前</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>X右  Y后</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>X左  Y后</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton_HMCdir_Ask">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>25</y>
      <width>60</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>查询</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_HMCdir_Set">
    <property name="geometry">
     <rect>
      <x>187</x>
      <y>25</y>
      <width>60</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>设置</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
